src/options.cpp

```diff
@@ -770,6 +771,32 @@ std::vector<std::pair<std::string, std::string>> options_manager::build_soundpac
     return soundpack_names;
 }
 
+std::vector<std::pair<std::string, std::string>> options_manager::build_renderer_list()
+{
+    std::vector<std::pair<std::string, std::string>> renderer_names;
+
+#ifdef TILES
+    int numRenderDrivers = SDL_GetNumRenderDrivers();
+    DebugLog( D_INFO, DC_ALL ) << "Number of render drivers on your system: " << numRenderDrivers;
+    for( int ii = 0; ii < numRenderDrivers; ii++ ){
+        SDL_RendererInfo ri;
+        SDL_GetRenderDriverInfo( ii, &ri );
+        DebugLog( D_INFO, DC_ALL ) << "Render driver: " << ii << "/" << ri.name;
+        renderer_names.emplace_back( ri.name, translate_marker( ri.name ) );
```

Comment:
The `translate_marker` here is pointless, it is only meaningful on string literals (like below), but not on variables.

Reply:
Removed it. Thanks.